src/app/campaign-info/campaign-info.component.html

```diff
@@ -1,21 +1,20 @@
 <div>
-  <!-- Todo: componentise this -->
   <biggive-campaign-highlights
     space-below=0
-    primary-figure-label="Amount raised"
+    [primaryFigureLabel]="primaryFigureLabel"
```

Comment:
I think introduce the property in the class probably makes things harder to follow, since then to understand this line you need to search for everywhere that writes to the primaryFigureLabel, and find there's the value set on assignment and the other value set in ngOnInit - and unless you search you don't know that there aren't more.

I think better to keep the object simpler and do this logic in the template:

```suggestion
    [primaryFigureLabel]="this.campaign.parentUsesSharedFunds ? 'Overall total raised' : 'Amount raised'"
```

Same applies to secondary figure label.